test/SE-13092.test.js

```diff
@@ -54,8 +54,8 @@ describe('SE-13092', () => {
         assert.typeOf(result, 'array');
         const item = result[0];
         const { value } = item;
-        assert.include(value, '<unknown-type>', 'uses unkown type name (opening tag)');
```

Comment:
Similarly to the previous comment. There is `unknown-type` hard coded into the code when the type cannot be determined. This means that this logic is not executed properly. This is something AMF changed and they said they will fix it. I won't change this tests or we will lose track of the issue.

Reply:
To be checked with AMF team